packages/types/src/organization.ts

```diff
@@ -51,7 +51,9 @@ export interface OrganizationResource extends ClerkResource {
   setLogo: (params: SetOrganizationLogoParams) => Promise<OrganizationResource>;
 }
 
-export type GetMembershipsParams = ClerkPaginationParams;
+export type GetMembershipsParams = {
+  role?: MembershipRole[];
```

Comment:
Why is this optional?

Reply:
You don't have to specify the role, If it is left empty or undefined it will return memberships of any role